x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/factories/utils/build_bulk_body.ts

```diff
@@ -51,7 +51,8 @@ export const buildBulkBody = (
   ignoreFields: ConfigType['alertIgnoreFields'],
   applyOverrides: boolean,
   buildReasonMessage: BuildReasonMessage,
-  indicesToQuery: string[]
+  indicesToQuery: string[],
+  alertTimestampOverride?: Date
```

Comment:
```suggestion
  alertTimestampOverride: Date | undefined
```
Let's make this required so we don't forget to pass it through in some calls to `buildBulkBody`. Right now the param is missing from the call in `wrapNewTermsAlerts`.